src/NUnitFramework/nunitlite/Options.cs

```diff
@@ -897,9 +897,7 @@ bool WriteOptionPrototype (TextWriter o, Option p, ref int written)
             }
 
             for ( i = GetNextOptionIndex (names, i+1);
-                    i < names.Length; i = GetNextOptionIndex (names, i+1)) {
-                Write (o, ref written, ", ");
-                Write (o, ref written, names [i].Length == 1 ? "-" : "--");
+                    i < names.Length; i = GetNextOptionIndep.OptionValueType is OptionValueType.Optional or OptionValueType.Requiredength == 1 ? "-" : "--");
```

Comment:
I'm confused, where did the `{` go? It's deleted in the red and not readded in the green.

Reply:
`Requiredength` also looks like it shouldn't compile.